packages/puppeteer/test/utils.ts

```diff
@@ -42,7 +49,7 @@ export function puppeteerOpts(): PuppeteerLaunchOptions {
   if (process.env.CI) {
     options.args = [];
     options.args.push('--no-sandbox', '--disable-setuid-sandbox');
-    options.executablePath = '/usr/bin/google-chrome-stable';
+    options.executablePath = process.env.CHROME_TEST_PATH;
```

Comment:
I think we probably want cases where the browser automation framework bundles a synced version of the browser (playwright, puppeteer) to use that synced version, so I don't think this should actually be going through BDM (and I don't think the original should have been going through the `/usr/bin/` version, either)